exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillCursor.java

```diff
@@ -260,6 +288,10 @@ void close() {
       // when the main thread is blocked waiting for the result.  In that case
       // we want to unblock the main thread.
       firstMessageReceived.countDown(); // TODO:  Why not call releaseIfFirst as used elsewhere?
+      //Stopping timeout clock
```

Comment:
since we are closing, do we need to care about the stopwatch?

Reply:
Just wrapping up any 'running' resources.

Reply:
Nothing is actually running in Stopwatch (it's just a state to indicate if elapsed time should use the current time or the time when Stopwatch was stopped...)

Reply:
Ok. Guess we'll do away with it. +1